{"Message":"發生錯誤。","ExceptionMessage":"在接收來自伺服器的要求時發生傳輸層級的錯誤。 (provider: TCP Provider, error: 0 - 信號等待逾時。)","ExceptionType":"System.Data.SqlClient.SqlException","StackTrace":"   於 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)\r\n   於 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   於 System.Data.SqlClient.TdsParserStateObject.ReadSniError(TdsParserStateObject stateObj, UInt32 error)\r\n   於 System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()\r\n   於 System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()\r\n   於 System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()\r\n   於 System.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)\r\n   於 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   於 System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()\r\n   於 System.Data.SqlClient.SqlDataReader.get_MetaData()\r\n   於 System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption)\r\n   於 System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)\r\n   於 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)\r\n   於 System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)\r\n   於 System.Data.SqlClient.SqlCommand.ExecuteScalar()\r\n   於 webapi2.Models.CchkUser.CheckToken(String guid) 於 E:\\303\\project\\專案\\司法院\\周邊系統\\OpenData_api\\JudJsonExporter_Elastic\\webapi2\\Models\\CchkUser.cs: 行 124\r\n   於 webapi2.Controllers.JDocController.Post(JDocParameter p) 於 E:\\303\\project\\專案\\司法院\\周邊系統\\OpenData_api\\JudJsonExporter_Elastic\\webapi2\\Controllers\\JDocController.cs: 行 38\r\n   於 lambda_method(Closure , Object , Object[] )\r\n   於 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_2.<GetExecutor>b__2(Object instance, Object[] methodParameters)\r\n   於 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 先前擲回例外狀況之位置中的堆疊追蹤結尾 ---\r\n   於 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   於 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   於 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 先前擲回例外狀況之位置中的堆疊追蹤結尾 ---\r\n   於 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   於 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   於 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 先前擲回例外狀況之位置中的堆疊追蹤結尾 ---\r\n   於 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   於 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   於 System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__15.MoveNext()","InnerException":{"Message":"發生錯誤。","ExceptionMessage":"信號等待逾時。","ExceptionType":"System.ComponentModel.Win32Exception","StackTrace":null}}